clients/fides-js/src/lib/initialize.ts

```diff
@@ -160,23 +155,10 @@ export const getInitialFides = ({
       cookie: FidesCookie;
       debug: boolean;
     }) => PrivacyExperience;
-  }): Partial<FidesGlobal> | null => {
-  const hasExistingCookie = !isNewFidesCookie(cookie);
-
-  // Register any configured consent migration providers
-  registerDefaultProviders(options);
-
-  // Check if there's consent available to migrate from any provider
-  const { consent: migratedConsent } = readConsentFromAnyProvider(options);
-  const hasMigratableConsent = !!migratedConsent;
```

Comment:
Were we doing this part twice before (`registerDefaultProviders` and `readConsentFromAnyProvider`)? Seems like it?

Reply:
yep! ...and only so we could form the `if` statement.